kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterService.java

```diff
@@ -136,7 +87,7 @@ public Map<String, Service> getSecondaryResources(
                 .getSecondaryResources(primary);
         return secondaryResources.stream()
                 .collect(Collectors.toMap(
-                        ClusterService::clusterName,
+                        (s) -> s.getMetadata().getName(), // TODO what should this map to? Used to be that service name == vcluster name, no longer
```

Comment:
the javadoc said it's an arbitrary string, I think as long as it matches the naming scheme from `desiredResources` so it can calculate the delta it's fine.

Reply:
Yes, this is my understanding.

Reply:
I'm not really following what's this bit is about.